### PR TITLE
EXTENSION: Allow Widgets to extend post-controls

### PR DESCRIPTION
**Pull request by Ahmedgagan**

### CODE REVIEW

**Comment by Ahmedgagan** [comment]
@jjaffeux 
@SamSaffron

File: app/assets/javascripts/discourse/app/widgets/post-menu.js [hunk 1]
```diff
@@ -565,6 +565,15 @@ export default createWidget("post-menu", {
       }
     }
 
+    const extraPostControls = applyDecorators(
+      this,
+      "extra-post-controls",
```
Comment (jjaffeux) [inline app/assets/javascripts/discourse/app/widgets/post-menu.js:570]: can we use: `after-post-controls` instead ?

Comment (Ahmedgagan) [reply app/assets/javascripts/discourse/app/widgets/post-menu.js:570]: As we want it inside the post-controls nav, so I think it's okay if we keep it `extra-post-controls`

**Comment by SamSaffron** [comment]
@jjaffeux are we good to merge this?

**Review by jjaffeux — APPROVED** [review]

**Comment by jjaffeux** [comment]
Yes I just merged it :+1:

**Pull request merged**